pool/pool.py

```diff
@@ -50,6 +50,7 @@
 from .singleton import create_absorb_transaction, get_singleton_state, get_coin_spend, get_farmed_height
 from .store.abstract import AbstractPoolStore
 from .store.sqlite_store import SqlitePoolStore
+from .store.mariadb_store import MariadbPoolStore
```

Comment:
Unused import, please remove.

Reply:
i believe that its needed if a db is supplied to pool_server.start_pool_server

Reply:
I needs to be done programatically either way, so whoever wants to use Mariadb will have to import it.

Unless someone adds support for selectable backend via config file ;)

Reply:
NOO youve forced my hand lol

Reply:
ill do this in a bit 
thanks for bringing it up

Reply:
added it @william-gr 
thanks for the suggestion

Reply:
let me know what you think

Reply:
also freenas is nice lol

Reply:
almost there

and thanks! about freenas/truenas :P